src/Admin/Pool.php

```diff
@@ -63,32 +57,28 @@ class Pool
     protected $titleLogo;
 
     /**
-     * @var array
+     * @var array<string, mixed>
      */
     protected $options = [];
 
     /**
-     * @var PropertyAccessorInterface
+     * @var PropertyAccessorInterface|null
      */
     protected $propertyAccessor;
 
     /**
-     * NEXT_MAJOR: change to TemplateRegistryInterface.
-     *
-     * @var MutableTemplateRegistryInterface
+     * @param array<string, mixed> $options
      */
-    private $templateRegistry;
```

Comment:
Seems like this should not be removed

Reply:
Yes, thanks. Not sure if I messed up something. Should I also replace MutableTemplateRegistryInterface with TemplateRegistryInterfac?  Is that comment is for 4.0 version (not sure I saw it before)